packages/augur-state/src/db/DB.ts

```diff
@@ -348,6 +351,28 @@ export class DB<TBigNumber> {
     return results.docs as unknown as Array<DisputeCrowdsourcerCompletedLog>;
   }
 
+    /**
+   * Queries the DisputeCrowdsourcerContribution DB
+   *
+   * @param {PouchDB.Find.FindRequest<{}>} request Query object
+   * @returns {Promise<Array<DisputeCrowdsourcerContributionLog>>}
+   */
+  public async findDisputeCrowdsourcerContributionLogs(request: PouchDB.Find.FindRequest<{}>): Promise<Array<DisputeCrowdsourcerContributionLog>> {
+    const results = await this.findInSyncableDB(this.getDatabaseName("DisputeCrowdsourcerContribution"), request);
+    return results.docs as unknown as Array<DisputeCrowdsourcerContributionLog>;
```

Comment:
How'd you figure out to do as twice? Whoa...never thought of that.

Reply:
Haha, I think Alex was the one who originally wrote this. I copied this from his original function to create these new find functions.